config/config.json

```diff
@@ -16,236 +16,249 @@
-        "cert-dcl54",
-        "cert-dcl58",
-        "cert-dcl59",
-        "cert-err09",
-        "cert-fio38",
-        "cert-oop11",
-        "cppcoreguidelines-special",
-        "google-build-namespaces",
-        "google-global",
-        "misc-assert",
-        "misc-bool",
-        "misc-dangling",
-        "misc-definitions",
-        "misc-fold",
-        "misc-forward",
-        "misc-forwarding",
-        "misc-inaccurate",
-        "misc-incorrect",
-        "misc-inefficient",
-        "misc-lambda",
-        "misc-macro",
-        "misc-misplaced",
-        "misc-move",
-        "misc-redundant",
-        "misc-sizeof",
-        "misc-string-compare",
-        "misc-string-constructor",
-        "misc-string-literal",
-        "misc-suspicious-enum",
-        "misc-suspicious-missing",
-        "misc-suspicious-semicolon",
-        "misc-swapped",
-        "misc-unconventional",
-        "misc-uniqueptr",
-        "misc-unused-raii",
-        "misc-use",
-        "misc-virtual",
-        "performance-inefficient-algorithm",
-        "performance-move",
-        "performance-noexcept"
-      ],
-      "sensitive": [
-        "boost",
-        "bugprone",
-        "cert-dcl03",
-        "cert-dcl16",
-        "cert-dcl21",
-        "cert-dcl50",
-        "cert-dcl54",
-        "cert-dcl58",
-        "cert-dcl59",
-        "cert-env33",
-        "cert-err09",
-        "cert-err52",
-        "cert-err58",
-        "cert-fio38",
-        "cert-flp30",
-        "cert-msc50",
-        "cert-msc51",
-        "cert-oop11",
-        "cppcoreguidelines-pro-type-static",
-        "cppcoreguidelines-slicing",
-        "cppcoreguidelines-special",
-        "google-build",
-        "google-explicit",
-        "google-global",
-        "google-runtime",
-        "misc-argument",
-        "misc-assert",
-        "misc-bool",
-        "misc-dangling",
-        "misc-definitions",
-        "misc-fold",
-        "misc-forward",
-        "misc-forwarding",
-        "misc-inaccurate",
-        "misc-incorrect",
-        "misc-inefficient",
-        "misc-lambda",
-        "misc-macro",
-        "misc-misplaced",
-        "misc-move",
-        "misc-multiple",
-        "misc-redundant",
-        "misc-sizeof",
-        "misc-string",
-        "misc-suspicious",
-        "misc-swapped",
-        "misc-unconventional",
-        "misc-uniqueptr",
-        "misc-unused-alias",
-        "misc-unused-raii",
-        "misc-unused-using",
-        "misc-use",
-        "misc-virtual",
-        "modernize",
-        "mpi",
-        "performance"
-      ],
-      "security": [
-        "cert"
-      ],
-      "portability": [
-        "bugprone-misplaced-widening",
-        "misc-misplaced-widening"
-      ],
-      "extreme": [
-        "boost",
-        "bugprone",
-        "cert",
-        "cppcoreguidelines",
-        "google",
-        "misc",
-        "modernize",
-        "mpi",
-        "performance"
-      ]
-    }
+    "clangsa": [
```

Comment:
Please update the documentation and describe this new format of config file: https://github.com/Ericsson/codechecker/blob/master/docs/analyzer/user_guide.md#checker-profiles.

Reply:
Good point! I always forget to update the documentation! 👍